testing/btest/cluster/websocket/tls-nocert.zeek

```diff
@@ -0,0 +1,143 @@
+# @TEST-DOC: Run a single node cluster (manager) with a websocket server that has TLS enabled without certs, similar to the Broker::listen_websocket() default.
+#
+# @TEST-REQUIRES: have-zeromq
+# @TEST-REQUIRES: python3 -c 'import websockets.asyncio'
+#
+# @TEST-GROUP: cluster-zeromq
+#
+# @TEST-PORT: XPUB_PORT
+# @TEST-PORT: XSUB_PORT
+# @TEST-PORT: LOG_PULL_PORT
+# @TEST-PORT: WEBSOCKET_PORT
+#
+# @TEST-EXEC: cp $FILES/zeromq/cluster-layout-simple.zeek cluster-layout.zeek
+# @TEST-EXEC: cp $FILES/zeromq/test-bootstrap.zeek zeromq-test-bootstrap.zeek
+#
+# @TEST-EXEC: zeek -b --parse-only manager.zeek
+# @TEST-EXEC: python3 -m py_compile client.py
+#
+# @TEST-EXEC: btest-bg-run manager "ZEEKPATH=$ZEEKPATH:.. && CLUSTER_NODE=manager zeek -b ../manager.zeek >out"
+# @TEST-EXEC: btest-bg-run client "python3 ../client.py >out"
+#
+# @TEST-EXEC: btest-bg-wait 30
+# @TEST-EXEC: btest-diff ./manager/out
+# @TEST-EXEC: btest-diff ./manager/.stderr
+# @TEST-EXEC: btest-diff ./client/out
+# @TEST-EXEC: btest-diff ./client/.stderr
+
+# @TEST-START-FILE manager.zeek
+@load ./zeromq-test-bootstrap
+redef exit_only_after_terminate = T;
+
+global ping_count = 0;
+
+global ping: event(msg: string, c: count) &is_used;
+global pong: event(msg: string, c: count) &is_used;
+
+event zeek_init()
+	{
+	Cluster::subscribe("/zeek/event/my_topic");
+
+	# Enable encryption, but don't require authentication (and not even certificates).
+	#
+	# This is the old Broker::listen_websocket() default behavior.
+	local tls_options = Cluster::WebSocketTLSOptions(
+		$cert_file="",
+		$key_file="",
+		$ca_file="NONE",
+		$ciphers="AECDH-AES256-SHA@SECLEVEL=0:AECDH-AES256-SHA:P-384",
+	);
+
+	local ws_server_options = Cluster::WebSocketServerOptions(
+		$listen_addr=127.0.0.1,
+		$listen_port=to_port(getenv("WEBSOCKET_PORT")),
+		$tls_options=tls_options,
+	);
+
+	Cluster::listen_websocket(ws_server_options);
+	}
+
+event ping(msg: string, n: count) &is_used
+	{
+	++ping_count;
+	print fmt("got ping: %s, %s", msg, n);
+	local e = Cluster::make_event(pong, "my-message", ping_count);
+	Cluster::publish("/zeek/event/my_topic", e);
+	}
+
+event Cluster::websocket_client_added(info: Cluster::EndpointInfo, subscriptions: string_vec)
+	{
+	print "Cluster::websocket_client_added", subscriptions;
+	}
+
+event Cluster::websocket_client_lost(info: Cluster::EndpointInfo, code: count, reason: string)
+	{
+	print "Cluster::websocket_client_lost";
+	terminate();
```

Comment:
If we did a clean `exit(0)` here we would expect nothing on stderr and wouldn't need to baseline it.

```suggestion
	exit(0);
```